app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid d-flex flex-column justify-content-center auth_container shadow-sm bg-body-tertiary rounded">
+  <h2>Log in</h2>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field field_input">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "username (email)" %>
+    </div>
+    <br>
+    <div class="field field_input">
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: "password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions text-center submit">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+  
+  <%= render "devise/shared/links" %>
```

Comment:
- [ ] As you stated, I checked the work and actually CSS is uploading properly. I suggest you go to the CSS file and delete the line 'width: 10%' and make 'login' look like a button and you are good to go.  
![image](https://github.com/Rafiduzzaman/blog-app/assets/91486791/f6071736-7d51-4c4e-96bf-c990804dba6f)